tests/test_logs.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from ddtrace.logs import TraceContextFilter
```

Comment:
We should be happy with this import. I guess it's fine to put it under a `logs` module.